documentation/guide-for-adding-banner-ads.md

```diff
@@ -0,0 +1,80 @@
+# Banner Ads Implementation Guide for all joomla.org sites
+This file describes how Joomla.org web masters should implement the ad serving code on our web properties.
+
+## ads.joomla.org
+We are running our own internal ([Revive](https://www.revive-adserver.com/)) ad server that should be serving all our ads. *We should not be serving ads directly from AdSense.*
+
+## Ad code
+The code to show ads is simple and in two parts.
+
+### General script
+```html
+<script async src="//ads.joomla.org/www/delivery/asyncjs.php"></script>
+```
+This code should be included **once** per page. The best way to include this code is most likely by pasting it into the template of the site. The code should be inside the `<body>` of the page near the `</body>` closing tag.
```

Comment:
Instead of encouraging to modify the template, why doesn't someone just make a module for this stuff then the module can be included where needed?  Don't have to worry about fighting with editor HTML cleanup or people copy/pasting the wrong stuff.